nmi_mysql/nmi_mysql.py

```diff
@@ -11,6 +11,7 @@
 
 CONFIG_KEYS = ['host', 'user', 'password', 'db', 'port']
 MAX_POOL_SIZE = 10
+MAX_RETRIES = 5
```

Comment:
MAX_RETRIES is used to catch large retry values (e.g. retry connecting 100 times which just does not make sense) or shouldn't we care about it?
The default retry value is 0. See this [line](https://github.com/pprmint/nmi_mysql/blob/cd00b1c89c7416617760ae79aaaf64b75802a71d/nmi_mysql/nmi_mysql.py#L104).